### PR TITLE
exporting projection layers of joiner separately for onnx

### PR DESCRIPTION
**Pull request by EmreOzkose**
1. Sherpa-onnx requires projection layers separately. Corresponding issue is [here](https://github.com/k2-fsa/sherpa-onnx/issues/7). 
2. "blank_id/unk_id/context_size" parameters are stored in all_in_one.onnx internally anymore.

### CODE REVIEW

File: egs/librispeech/ASR/pruned_transducer_stateless3/export.py [hunk 1]
```diff
@@ -68,6 +68,7 @@
     - encoder.onnx
     - decoder.onnx
     - joiner.onnx
+    - all_in_one.onnx
```
Comment (csukuangfj) [inline egs/librispeech/ASR/pruned_transducer_stateless3/export.py:71]: Please change `three` on line 65 to `four` or just remove it.

File: egs/librispeech/ASR/pruned_transducer_stateless3/export.py [hunk 2]
```diff
@@ -485,16 +487,13 @@ def export_joiner_model_onnx(
...
     """
     encoder_out_dim = joiner_model.encoder_proj.weight.shape[1]
     decoder_out_dim = joiner_model.decoder_proj.weight.shape[1]
-    encoder_out = torch.rand(1, encoder_out_dim, dtype=torch.float32)
-    decoder_out = torch.rand(1, decoder_out_dim, dtype=torch.float32)
+    encoder_out = torch.rand(1, 1, 1, encoder_out_dim, dtype=torch.float32)
+    decoder_out = torch.rand(1, 1, 1, decoder_out_dim, dtype=torch.float32)
 
-    project_input = True
+    project_input = False
```
Comment (csukuangfj) [inline egs/librispeech/ASR/pruned_transducer_stateless3/export.py:496]: Could you also update
- https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/pruned_transducer_stateless3/onnx_check.py
- https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/pruned_transducer_stateless3/onnx_pretrained.py

**Comment by EmreOzkose** [comment]
onnx_check argument problem in actions is fixed.

**Comment by csukuangfj** [comment]
> onnx_check argument problem in actions is fixed.

Thanks! Will look into it later today.

**Comment by csukuangfj** [comment]
Thanks!

**Pull request merged**